.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.18.0.cjs"
```

Comment:
I'm pretty interested in knowing what the bug that required this was.

The pattern of committing the whole yarn program to source control still feels very weird to me but I guess it is the advised way to do it, and we've not had issues with the approach in other repos (like `employee-imports`). 

I'm just curious about the bug.

Reply:
I just removed this again to see if it works in CI. iirc It was an issue with `yarn-audit-fix` where it was unable to resolve something in anything other than 1.18.0 and lower.